packages/eslint-config-airbnb/.eslintrc

```diff
@@ -3,6 +3,6 @@
   "rules": {
     // disable requiring trailing commas because it might be nice to revert to
     // being JSON at some point, and I don't want to make big changes now.
-    "comma-dangle": 0,
-  },
+    "comma-dangle": 0
+  }
```

Comment:
eslintrc isn't JSON, it's an extended format that allows trailing commas. these are here on purpose.